core/src/main/java/overflowdb/NodeRef.java

```diff
@@ -55,8 +55,8 @@ public boolean isCleared() {
     return node == null;
   }
 
-  /* only called by @ReferenceManager */
-  protected void clear() {
+  /* only supposed to be called by @NodesWriter */
+  public void clear() {
```

Comment:
Can we make this package-private? Or alternatively make it a static method (i.e. `static void clearNode(NodeRef ref)`)?

Point is to avoid cluttering the REPL completion in ocular/joern

Reply:
Good point, we don't really want users to see this. 
`NodesWriter`is in the `storage` subpackage.... I'm debating with myself if that should move or we should go for your static method suggestion.... I think I prefer the latter

Reply:
done in 3ff21af